metricbeat/metricbeat.full.yml

```diff
@@ -284,6 +284,9 @@ metricbeat.modules:
   # Redis hosts
   #hosts: ["127.0.0.1:6379"]
 
+  # Password to authenticate to the hosts
```

Comment:
The password is already available on line 308. I think it needs to be added to the docs instead.